gbus/saga/instance.go

```diff
@@ -34,6 +34,9 @@ type Instance struct {
 	StartedByMessageID string
 	//StartedByRPCID the rpc id of the message that created the saga
 	StartedByRPCID string
+
+	// the time.Now() timestamp when the saga was created
+	CreatedAt time.Time
```

Comment:
this field needs to be persistent. you should a db migration for it

Reply:
In addition, documentation in go begin with the member you are documenting, should be:
//CreatedAt the time.Now() timestamp when the saga was created